docs/content/direct/common-setup-kind.md

```diff
@@ -0,0 +1,100 @@
+# KubeStellar Core Sample Setup -- Kind with OCM
+
+## Prereqs
+
+1. Make sure you have all the prerequisite software installed [pre-reqs](pre-reqs.md).
+
+1. If you ran through a setup (for one of our example scenarios) previously then you will need to do a bit of cleanup first. See how it is done in the cleanup script for our E2E tests (in `test/e2e/common/cleanup.sh`).
+
+## Setting up the KubeStellar Core components
+
+The following steps establish the setup in our [example scenarios](examples.md) and are a good starting point.
+
+### Prepare the environment ###
```

Comment:
What is the trailing `###` here for?

Reply:
copied from something else. I inferred it was the equivalent of an html header close tag. My mistake